core/iwasm/interpreter/wasm.h

```diff
@@ -273,28 +273,14 @@ typedef struct WASMType {
     struct WASMType *parent_type;
     uint32 parent_type_idx;
 
-    uint32 data[1];
+    uint32 data[0];
```

Comment:
How about removing this line? It is unused and may cause compile warning for some compilers.

Reply:
OK